ReactiveXComponent/WebSocket/WebSocketMessageHelper.cs

```diff
@@ -64,6 +64,25 @@ public static string SerializeRequest(string requestKey, WebSocketEngineHeader h
             return $"{beforeJson} {jsonContent}{Environment.NewLine}";
         }
 
+        public static string SerializeXCApiRequest(object message, string data = null)
+        {
+            if (message == null)
+                return $"{string.Empty}{Environment.NewLine}";
+
+            var jsonMessage = new StringBuilder().Append(message);
+
+            if (string.IsNullOrEmpty(data))
+            {
+                jsonMessage.Append(" {}");
+            }
+            else
+            {
+                jsonMessage.Append(" {\"Name\": \"").Append(data).Append("\"}");

```

Comment:
use some high level API to build Json documents! instead of building them by hand!